# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/boost/hotfix-move-detai.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

https://github.com/clayne/boost-move/commit/5f073f8f00ee23b4502c0ad30a3aa2a5154cd1e8

--- a/boost/move/detail/std_ns_begin.hpp
+++ b/boost/move/detail/std_ns_begin.hpp
@@ -9,6 +9,7 @@
 #//
 #//////////////////////////////////////////////////////////////////////////////
 #
+#include <boost/config.hpp>
 #if defined(_LIBCPP_VERSION)
    #if defined(__clang__)
       #define BOOST_MOVE_STD_NS_GCC_DIAGNOSTIC_PUSH
@@ -26,6 +27,9 @@
 #elif defined(BOOST_DINKUMWARE_STDLIB)
    #define BOOST_MOVE_STD_NS_BEG _STD_BEGIN
    #define BOOST_MOVE_STD_NS_END _STD_END
+#else
+   #define BOOST_MOVE_STD_NS_BEG namespace std{
+   #define BOOST_MOVE_STD_NS_END
 #endif
 
 #if defined(_MSC_VER) && (_MSC_VER >= 1915)
